.gitignore

```diff
@@ -8,6 +7,10 @@
 .AppleDouble
 .LSOverride
 
+# Static code
+/docs/css/
+/docs/js/
```

Comment:
Since the files in here are being updated by gulp, there really isn't any need to store them, etc. What matters is the files they are editing.